src/redux/ConfigureStore.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, combineReducers } from 'redux';
+import booksReducers from './Books/Books';
+import categoriesReducer from './Categories/Categories';
+
+const rootReducer = combineReducers({
+  booksReducers,
+  categoriesReducer,
+});
+
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable max-len */
```

Comment:
- [ ] Please let's uncomment and rather fix the issue. This way, Linters rules are not altered.

Reply:
Thank you for your review. I have solved this issue.